test/e2e-common.sh

```diff
@@ -111,8 +111,8 @@ function install_latest_release() {
     fail_test "Knative latest release installation failed"
 }
 
-function run_postinstall() {
-  ko apply --strict -f ${POST_INSTALL_V015} || return 1
+function run_preinstall_V016() {
```

Comment:
did you mean to call this on L57 of the test?

Reply:
@vaikas Oh boy :facepalm: